pkg/plugins/policies/meshhttproute/api/v1alpha1/validation.go

```diff
@@ -218,3 +220,46 @@ func validateFilters(filters *[]Filter, matches []Match) validators.ValidationEr
 
 	return errs
 }
+
+func validateHeaderModifier(modifier *HeaderModifier) validators.ValidationError {
+	var errs validators.ValidationError
+
+	counter := 0
+	if modifier.Set != nil {
+		counter++
+	}
+	if modifier.Add != nil {
+		counter++
+	}
+	if modifier.Remove != nil {
+		counter++
+	}
+
+	if counter != 1 {
+		errs.AddViolationAt(validators.Root(), validators.MustHaveOnlyOne("headerModifier", "set", "add", "remove"))
```

Comment:
Why is this a problem? I think we only need to validate for multiple actions for the same header name.

Reply:
Right, I didn't pay attention to "per header name" in the comment. I'll update validation